system/netdata.service.in

```diff
@@ -44,5 +44,21 @@ CPUSchedulingPolicy=idle
 # For scheduling policy 'other' and 'batch', this sets the lowest niceness of netdata (-20 highest to 19 lowest).
 #Nice=0
 
+# Capabilities
+CapabilityBoundingSet=CAP_DAC_OVERRIDE    # is required for freeipmi and slabinfo plugins
+CapabilityBoundingSet=CAP_DAC_READ_SEARCH # is required for apps plugin
+CapabilityBoundingSet=CAP_FOWNER          # is required for freeipmi plugin
+CapabilityBoundingSet=CAP_SETPCAP         # is required for apps, perf and slabinfo plugins
+CapabilityBoundingSet=CAP_SYS_ADMIN       # is required for perf plugin
+CapabilityBoundingSet=CAP_SYS_PTRACE      # is required for apps plugin
+CapabilityBoundingSet=CAP_NET_RAW         # is required for fping app
+
+# Sandboxing
+ProtectSystem=full
+ProtectHome=read-only
+PrivateTmp=true
```

Comment:
Unless I'm misunderstanding how `PrivateTmp=true` works (and I might be as I don't really use systemd myself), this will actually break `netdatacli` functionality. See https://github.com/netdata/netdata/issues/7587 for details, until that's resolved we can't use `PrivateTmp=true` in our upstream unit files.

Reply:
Disabled PrivateTmp.